Tests/L0/Gradle/_suite.ts

```diff
@@ -597,4 +612,88 @@ describe('gradle Suite', function() {
             });
     })
 
+    it('Gradle with SonarQube - Should run Gradle with all default inputs when SonarQube analysis disabled', function(done) {
+        // Arrange
+        setResponseFile('gradleGood.json');
+
+        var tr = new TaskRunner('gradle', true, true);
+        tr = setDefaultInputs(tr);
+        tr.setInput('sqAnalysisEnabled', 'false');
+
+        // Act
+        tr.run()
+            .then(() => {
+                // Assert
+                assert(tr.succeeded, 'task should have succeeded');
+                assert(tr.invokedToolCount == 1, 'should have only run gradle 1 time');
+                assert(tr.resultWasSet, 'task should have set a result');
+                assert(tr.stderr.length == 0, 'should not have written to stderr');
+                assert(tr.ran('gradlew build'), 'it should have run only the default settings');
+                done();
+            })
+            .fail((err) => {
+                console.log(tr.stdout);
+                done(err);
+            });
+    });
+
+    it('Gradle with SonarQube - Should run Gradle with SonarQube analysis', function(done) {
+        // Arrange
+
+        setResponseFile('gradleGood.json');
+
+        var tr = new TaskRunner('gradle', true, true);
```

Comment:
It turns out turning the third argument to TRUE means we don't need the dynamic response.json fine any more. Some experimentation required - maybe the Maven tasks could also benefit from this.